### ISSUE TITLE
[REQ] Peloton target metrics in a range/interval 

### ISSUE THREAD

**Issue by blautob** [cagnulein/qdomyos-zwift#1292]
Option to set the Peloton target Resistance and target Cadence to a range e.g. if the instructors tells between 90-110 and resistance 55-65 it would be great to see it in a range interval on the tile like "90-110" and "55-65" as i believe this is easier/more natural to follow

**Additional context**

see example like done in the bookmarklet @ https://github.com/ZV3zrmCT/peloton_schwinn_metrics

**Comment by cagnulein** [comment]
@blautob I was starting this but I saw that the range it's already visibile in the target resistance/cadence tiles in the second line. Isn't it enough?

**Comment by blautob** [comment]
Yes you are right it is there in very small font. For me it would be do muchh better to have it in the bold numbers e. Cadence „80-90“ omitting the small numbers and text below. 
![E02B6C42-021B-42B6-8EFA-B3881B7F5523](https://user-images.githubusercontent.com/24385671/220891646-7ec7a27c-c287-4a7d-8d69-9e5f5b0c6e90.jpeg)

**Comment by cagnulein** [comment]
Ok I will keep this in low priority, thanks
